From bf017ba46e9e3c3ef71b77a228315f755e121995 Mon Sep 17 00:00:00 2001
From: Georgii Staroselskii <georgii.staroselskii@emlid.com>
Date: Sun, 25 Mar 2018 17:54:40 +0300
Subject: [PATCH] arm: sunxi: enable BOOTZ command

Enable it just in case we might need it in the future
---
 configs/emlid_neutis_n5_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/emlid_neutis_n5_defconfig b/configs/emlid_neutis_n5_defconfig
index 66244b6..a4998ae 100644
--- a/configs/emlid_neutis_n5_defconfig
+++ b/configs/emlid_neutis_n5_defconfig
@@ -9,6 +9,7 @@ CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_SPL=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_CMD_BOOTZ=y
 # CONFIG_CMD_IMLS is not set
 # CONFIG_CMD_FLASH is not set
 # CONFIG_CMD_FPGA is not set
-- 
2.7.4

